### PR TITLE
add Ubuntu Advantage shop to global nav

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* updated global nav to:
 * remove Discourse
 * replace the link to shop to buy.ubuntu.com
 * add new merchandise link in More
 * remove Apps link as that site is no longer supported
* removed accessibility-aide with list of things to do on forms as they are out of date and not used
* replaces urls for shop to buy in a few places
* removed old header.html file

## QA

* see that the new nav is loading and correct
* see that there are no links to the old shop

## Issue / Card

[trello card](https://trello.com/c/h2ZF0lon)

### CODE REVIEW

**Comment by pmahnke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;h2ZF0lon&#x2F;1451-change-global-nav-links-for-the-ubuntu-advantage-shop"> Change global nav links for the Ubuntu Advantage shop</a>

File: templates/desktop/contact-us.html [hunk 1]
```diff
@@ -5,87 +5,87 @@
...
+        <li><a class="external" href="http://askubuntu.com/">Ask Ubuntu</a></li>
+        <li><a class="external" href="http://ubuntuforums.org/">The Ubuntu forum</a></li>
+        <li><a class="external" href="https://help.ubuntu.com/">Help docs</a></li>
+    </ul>
+</div>
+<div class="four-col box last-col">
+    <h3>Looking for a little professional support?</h3>
+    <p>If you are a small organisation, you can purchase packs of Ubuntu Advantage in our shop.</p>
+    <p><a href="https://buy.ubuntu.com/">Visit the Ubuntu Advantage shop&nbsp;&rsaquo;</a></p>
+</div>
```
Comment (anthonydillon) [inline templates/desktop/contact-us.html]: You seem to have lost the tabbing here

File: templates/desktop/contact-us.html [hunk 2]
```diff
@@ -5,87 +5,87 @@
 {% block meta_keywords %}Ubuntu, Ubuntu operating system, OS, Linux, Windows alternative, Mac alternative, thin client, Business desktop{% endblock %}
 
 {% block second_level_nav_items %}
-	{% include "templates/_nav_breadcrumb.html" with section_title="Desktop" page_title="Contact us" %}
+{% include "templates/_nav_breadcrumb.html" with section_title="Desktop" page_title="Contact us" %}
```
Comment (anthonydillon) [inline templates/desktop/contact-us.html]: Missing tab here

**Comment by anthonydillon** [comment]
Code looks good with some missing tabbing :+1:

**Comment by anthonydillon** [comment]
QA ok :+1:

**Pull request merged**